### PR TITLE
Add batch support

### PR DESCRIPTION
**Pull request by bholmesdev**
## Changes

Add support for `db.batch()` to run multiple queries in a single transaction.
- Bump Drizzle to 0.29.5 (minor bump)
- Implement batching for the remote adapter. Nothing to do for local

## Testing

- Add `db.batch()` call to basics test fixture

## Docs

N/A

### CODE REVIEW

**Comment by bholmesdev** [comment]
!preview db-batch

File: packages/db/package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
     "@libsql/client": "^0.4.3",
     "async-listen": "^3.0.1",
     "deep-diff": "^1.0.2",
-    "drizzle-orm": "^0.28.6",
+    "drizzle-orm": "0.29.5",
```
Comment (delucis) [inline packages/db/package.json:65]: Any specific notes as to why this is pinned to an exact version?

**Review by delucis — COMMENTED** [review]
I’m not familiar with the code, so hard to review, but left one question.

In the PR description docs says “N/A” but if this is something we support, wouldn’t we need to document it?

File: packages/db/package.json (hunk 1 above)
Comment (bholmesdev) [reply packages/db/package.json:65]: Ah, it shouldn't be pinned! Good catch

**Comment by bholmesdev** [comment]
@delucis re-documentation: I'm using Sarah's suggestion to outsource to the Drizzle docs as much as possible to avoid explaining all of their features. There _is_ an argument to document `db.batch()` as a best practice for seeding though. There were mixed thoughts on recommending batch early, so I'll raise in the docs discussion thread.

**Review by delucis — APPROVED** [review]
> I'm using Sarah's suggestion to outsource to the Drizzle docs

Ah, OK — assumed as _we_ were adding support, there might be something Astro-specific to the API separate from Drizzle’s batch API. If that’s not the case, then all’s good!

Approving from a place of ignorance — a `batch` made in heaven 🙌

**Pull request merged**